tests/sqllogictests/suites/ee/04_ee_inverted_index/04_0000_inverted_index_base.test

```diff
@@ -427,6 +427,41 @@ onlyif mysql
 statement error 1105
 SELECT id, score(), title FROM books WHERE query('title:(设计 实现)^5 description:(学习 +神经网络)^1.1') ORDER BY score() DESC
 
+statement ok
+CREATE TABLE t1 (id int, body json)
+
+statement ok
+CREATE INVERTED INDEX IF NOT EXISTS idx ON t1(body) tokenizer = 'chinese'
```

Comment:
Can we only create the inverted index for a json item?
Like:
```
CREATE OR REPLACE INVERTED INDEX idx ON t1(body.title);
```

Reply:
This PR doesn't support this case, we can add it later.